src/main/java/org/broadinstitute/hellbender/tools/exome/TargetTableColumn.java

```diff
@@ -11,10 +11,10 @@
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
  */
 public enum TargetTableColumn {
-    NAME("name", true),
-    CONTIG("contig", true),
-    START("start", true),
-    END("stop", true),
+    NAME("name", true), /* it was a terrible idea to change this from "NAME" to "name"! */
+    CONTIG("contig", true), /* it was a terrible idea to change this from "CONTIG" to "name"! */
```

Comment:
updated the error in my rant! also, I would like to keep the rant there to make sure we don't do this in the future! :D